library/asb_encode_binding.py

```diff
@@ -22,6 +22,16 @@
       - 'dictionary of key/value pairs to encode for a binding.  Keys will become the injected environment variables.'
     required: true
     default: {}
+ env:
+        - Set via the downward API on the APB Pod
+        - name: ENV_NAME
```

Comment:
Should this be POD_NAME?

Reply:
I took these from the other module in the library asb_encode_binding to keep them consistent: https://github.com/ansibleplaybookbundle/ansible-asb-modules/blob/master/library/asb_encode_binding.py#L57